api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package org.pmiops.workbench.google;
+
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import javax.inject.Provider;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.google.Utils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CloudStorageServiceImpl implements CloudStorageService {
+
+  final Provider<WorkbenchConfig> configProvider;
+
+  @Autowired
+  public CloudStorageServiceImpl(Provider<WorkbenchConfig> configProvider) {
+    this.configProvider = configProvider;
+  }
+
+  public String readInvitationKey() {
+    String bucketName = configProvider.get().googleCloudStorageService.credentialsBucketName;
+    return readToString(bucketName, "invitation-key.txt").trim();
```

Comment:
We could cache the invitation key, FWIW. (Either way seems OK, given that we only read this on account creation.)